tests/scattering2d/test_jax_scattering2d.py

```diff
@@ -0,0 +1,285 @@
+import os
+import io
+import jax.numpy as jnp
+import numpy as np
+from kymatio import Scattering2D
+from collections import namedtuple
+import pytest
+
+
+backends = []
+
+from kymatio.scattering2d.backend.jax_backend import backend
+backends.append(backend)
+
+
+class TestPad:
+    @pytest.mark.parametrize('backend', backends)
+    def test_Pad(self, backend):
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=False)
+
+        x = np.random.randn(4, 4) + 1J * np.random.randn(4, 4)
+        x = x[np.newaxis, ...]
+
+        z = pad(x)
+
+        assert z.shape == (1, 8, 8)
+        assert z[0, 2, 2] == x[0, 0, 0]
+        assert z[0, 1, 0] == x[0, 1, 2]
+        assert z[0, 1, 1] == x[0, 1, 1]
+        assert z[0, 1, 2] == x[0, 1, 0]
+        assert z[0, 1, 3] == x[0, 1, 1]
+
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=True)
+
+        x = np.random.randn(8, 8) + 1J * np.random.randn(8, 8)
+
+        z = pad(x)
+
+        assert np.allclose(x, z)
```

Comment:
this doesn't seem like it can pass

Reply:
it passes

Reply:
how? `z=pad(x)` can't even be of the same shape as `x`

Reply:
ah, it's `pre_pad=False` -- does this mean it's already padded or not? Not, right? So then they should be different shapes. I guess we wanted to get rid of that `pre_pad` thing anyway